web/src/pages/apd/activities/cost-allocation/CostAllocation/OtherFunding.js

```diff
@@ -21,28 +23,66 @@ import {
 import { t } from '../../../../../i18n';
 import RichText from '../../../../../components/RichText';
 
+import Joi from 'joi';
+
+const otherSourcesSchema = Joi.object({
```

Comment:
can you move this to the schemas folder in common

Reply:
Right!